app/src/main/res/values-hdpi/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <integer name="profilechooser_span">4</integer>
```

Comment:
Do we want to use a different span count for portrait versus landscape?

Reply:
@BenHenning Ya it's advised to use 2 as span count for portrait mode.

Reply:
For all display densities? Can you make sure the tests verify this portrait vs. landscape span count? It didn't seem like they did before.

Reply:
Okay.